files/templates/alertmanager/alertmanager.yaml

```diff
@@ -429,5 +429,11 @@ inhibit_rules:
 - source_matchers:
     - vintage_cluster_release_before_v19=true
   target_matchers:
-    - cancel_if_vintage_cluster_release_before_v19=true
+    - cancel_if_cluster_is_not_running_prometheus_agent=true
+  equal: [cluster_id]
+
+- source_matchers:
+    - capi_cluster_prometheus_agent=true
```

Comment:
So this should be capi_cluster_prometheus_agent=false right? Hence my last comment on the other PR :)